CP77.CR2W/Modkit/Uncook.cs

```diff
@@ -128,87 +128,85 @@ public static partial class ModTools
         public static bool Uncook(Stream cr2wStream, FileInfo cr2wFileName, string ext, 
             EUncookExtension uncookext = EUncookExtension.dds, bool flip = false)
         {
-            
+            if (!Enum.GetNames(typeof(ECookedFileFormat)).Contains(ext))
+                return GenerateBuffers(cr2wStream, cr2wFileName);
+
+            if (!Enum.TryParse(ext, true, out ECookedFileFormat extAsEnum))
+                return false;
 
-            if (Enum.GetNames(typeof(ECookedFileFormat)).Contains(ext))
+            // read the cr2wfile
+            using var br = new BinaryReader(cr2wStream);
+            var cr2w = TryReadCr2WFile(br);
+            if (cr2w == null)
             {
-                if (!Enum.TryParse(ext, true, out ECookedFileFormat extAsEnum))
-                    return false;
+                Logger.LogString($"Failed to read cr2w file {cr2wFileName.FullName}", Logtype.Error);
+                return false;
+            }
+            cr2w.FileName = cr2wFileName.FullName;
 
-                // read the cr2wfile
-                using var br = new BinaryReader(cr2wStream);
-                var cr2w = TryReadCr2WFile(br);
-                if (cr2w == null)
-                {
-                    Logger.LogString($"Failed to read cr2w file {cr2wFileName.FullName}", Logtype.Error);
-                    return false;
-                }
-                cr2w.FileName = cr2wFileName.FullName;
+            // decompress buffers
+            var buffers = GetBuffers(cr2w, br);
 
-                // decompress buffers
-                var buffers = GetBuffers(cr2w, br);
 
+            // uncook textures, meshes etc
+            return extAsEnum switch
+            {
+                ECookedFileFormat.mesh => GenerateBuffers(cr2wStream, cr2wFileName),
```

Comment:
we should add here the actual mesh uncooking: conversion to fbx/glb. 
until that is done, simply extract buffers normally.